### PR TITLE
chore: repairing taxonomies

### PR DESCRIPTION
**Pull request by aleene**
### What
various fixes

### Related issue(s) and discussion
- Fixes #10079

### CODE REVIEW

File: taxonomies/food/categories.txt [hunk 1]
```diff
@@ -4297,7 +4297,7 @@ origins:en: en:France, fr:Bourgogne
 fr:Eau-de-vie de vin de la Marne
 de:Branntwein aus Marne, Branntwein aus dem Marne Tal
 nl:Wijn-eau-de-vies uit de Marne
-origins:en: en:France, fr:Marne
```
Comment (alexgarel) [inline taxonomies/food/categories.txt:4300]: fr:Marne not existing is only a warning, so I don't know if we should remove those.

@stephanegigandet and @teolemon what do you think ?

**Review by alexgarel — COMMENTED** [review]
Incredible work ! :bookmark_tabs: :feather:

I wait for a deliberation on origins before approval !

File: taxonomies/labels.txt [hunk 2]
```diff
@@ -3791,7 +3790,7 @@ label_categories:en: en:Geographic label
 
 fr:Saveurs en Or, Saveurs en'Or
 wikidata:en:Q3474531
-origins:en: en:france, fr:Hauts-de-France
+origins:en: en:france
```
Comment (teolemon) [inline taxonomies/labels.txt:3793]: normally, that's a label for Hauts-de-France @TaciteOFF

File: taxonomies/food/categories.txt (hunk 1 above)
Comment (alexgarel) [reply taxonomies/food/categories.txt:4300]: With @teolemon we think we should not remove this, but maybe, in another PR, add those to origins.txt

Comment (stephanegigandet) [reply taxonomies/food/categories.txt:4300]: Agreed, it would be best to add them to origins.txt

File: taxonomies/food/categories.txt [hunk 3]
```diff
@@ -4114,7 +4114,7 @@ fr:Armagnacs, Armagnac
 nl:Armagnacs, Armagnac
 pl:Armagnac, Armaniak
 wikidata_category:en:Q9469550
-origins:en: en:France, fr:Gascogne
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:4117]:
```suggestion
origins:en: en:France, fr:Gascogne
```

File: taxonomies/food/categories.txt [hunk 4]
```diff
@@ -4128,7 +4128,7 @@ en:Armagnac-Ténarèze
 de:Armagnac aus dem Anbaugebiet Ténarèze
 fr:Armagnac-Ténarèze
 nl:Armagnac uit Ténarèze
-origins:en: en:France, fr:Gascogne
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:4131]:
```suggestion
origins:en: en:France, fr:Gascogne
```

File: taxonomies/food/categories.txt [hunk 5]
```diff
@@ -4138,7 +4138,7 @@ de:Armagnac aus dem Anbaugebiet Bas-Armagnac
 fr:Bas-Armagnac
 wikidata:en:Q2201024
 wikipedia:en:https://en.wikipedia.org/wiki/Bas-Armagnac
-origins:en: en:France, fr:Gascogne
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:4141]:
```suggestion
origins:en: en:France, fr:Gascogne
```

File: taxonomies/food/categories.txt [hunk 6]
```diff
@@ -4244,7 +4244,7 @@ de:Apfelbrand aus Maine
 nl:Eau-de-vies uit de Maine-streek
 protected_name_file_number:en: PGI-FR-02036
 protected_name_type:en: gi
-origins:en: en:France, fr:Maine
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:4247]:
```suggestion
origins:en: en:France, fr:Maine
```

File: taxonomies/food/categories.txt [hunk 7]
```diff
@@ -4262,7 +4262,7 @@ ciqual_food_name:fr:Eau de vie de vin, type armagnac, cognac
 fr:Eau-de-vie de Jura
 de:Branntwein aus dem Jura
 nl:Eau-devies uit de Jura
-origins:en: en:France, fr:Jura
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:4265]:
```suggestion
origins:en: en:France, fr:Jura
```

File: taxonomies/food/categories.txt [hunk 8]
```diff
@@ -4283,7 +4283,7 @@ origins:en: en:France, fr:Normandie
 fr:Eau-de-vie de poiré du Maine
 de:Birnenbrand aus Maine
 nl:Peren-eau-de-vies uit de Maine-streek
-origins:en: en:France, fr:Maine
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:4286]:
```suggestion
origins:en: en:France, fr:Maine
```

File: taxonomies/food/categories.txt [hunk 9]
```diff
@@ -4297,7 +4297,7 @@ origins:en: en:France, fr:Bourgogne
 fr:Eau-de-vie de vin de la Marne
 de:Branntwein aus Marne, Branntwein aus dem Marne Tal
 nl:Wijn-eau-de-vies uit de Marne
-origins:en: en:France, fr:Marne
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:4300]:
```suggestion
origins:en: en:France, fr:Maine
```

File: taxonomies/food/categories.txt [hunk 10]
```diff
@@ -4315,7 +4315,7 @@ de:Branntwein aus dem Rhône Tal, Branntwein aus Côtes-du-Rhône
 nl:Wijn-eau-de-vies uit de Côtes-du-Rhône
 protected_name_file_number:en: PGI-FR-02070
 protected_name_type:en: gi
-origins:en: en:France, fr:Côtes-du-Rhône
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:4318]:
```suggestion
origins:en: en:France, fr:Côtes-du-Rhône
```

File: taxonomies/food/categories.txt [hunk 11]
```diff
@@ -4343,14 +4343,14 @@ origins:en: en:France, fr:Provence
 fr:Eau-de-vie de vin originaire des Coteaux de la Loire
 de:Branntwein aus dem Loire Tal, Branntwein aus Coteaux de la Loire
 nl:Wijn-eau-de-vies uit de Coteaux de la Loire
-origins:en: en:France, fr:Coteaux de la Loire
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:4346]:
```suggestion
origins:en: en:France, fr:Coteaux de la Loire
```

File: taxonomies/food/categories.txt [hunk 12]
```diff
@@ -4343,14 +4343,14 @@ origins:en: en:France, fr:Provence
...
 #wikidata:en:
 
 <en:French Eaux-de-vie
 fr:Eau-de-vie de vin originaire du Bugey
 de:Branntwein aus dem Bugey
 nl:Wijn-eau-de-vies uit de Bugey
-origins:en: en:France, fr:Bugey
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:4353]:
```suggestion
origins:en: en:France, fr:Bugey
```

File: taxonomies/food/categories.txt [hunk 13]
```diff
@@ -4369,14 +4369,14 @@ de:Branntwein aus dem Languedoc
 nl:Wijn eau-de-vies uit de Languedoc
 protected_name_file_number:en: PGI-FR-02067
 protected_name_type:en: gi
-origins:en: en:France, fr:Languedoc
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:4372]:
```suggestion
origins:en: en:France, fr:Languedoc
```

File: taxonomies/food/categories.txt [hunk 14]
```diff
@@ -4369,14 +4369,14 @@ de:Branntwein aus dem Languedoc
...
 #wikidata:en:
 
 <en:French Eaux-de-vie
 fr:Eau-de-vie des Charentes
 de:Branntwein aus Charente
 nl:Eau-de-vies uit de Charante
-origins:en: en:France, fr:Charentes
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:4379]:
```suggestion
origins:en: en:France, fr:Charentes
```

File: taxonomies/food/categories.txt [hunk 15]
```diff
@@ -4632,7 +4632,7 @@ origins:en: en:France
 fr:Rhum des Antilles françaises
 protected_name_file_number:en: PGI-FR-01990
 protected_name_type:en: gi
-origins:en: en:France, fr:Antilles françaises
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:4635]:
```suggestion
origins:en: en:France, fr:Antilles françaises
```

File: taxonomies/food/categories.txt [hunk 16]
```diff
@@ -4666,7 +4666,7 @@ origins:en: en:France, fr:Guadeloupe
 fr:Rhum de la Guyane
 protected_name_file_number:en: PGI-FR-02066
 protected_name_type:en: gi
-origins:en: en:France, en:French Guyana
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:4669]:
```suggestion
origins:en: en:France, en:French Guyana
```

File: taxonomies/food/categories.txt [hunk 17]
```diff
@@ -5635,7 +5635,7 @@ fr:Crèmes de cassis françaises
 fr:Cassis de Saintonge
 protected_name_file_number:en: PGI-FR-01986
 protected_name_type:en: gi
-origins:en: en:France, fr:Saintonge
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:5638]:
```suggestion
origins:en: en:France, fr:Saintonge
```

File: taxonomies/food/categories.txt [hunk 18]
```diff
@@ -5649,7 +5649,7 @@ origins:en: en:France, fr:Bourgogne
 fr:Cassis de Dijon
 protected_name_file_number:en: PGI-FR-01991
 protected_name_type:en: gi
-origins:en: en:France, fr:Dijon
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:5652]:
```suggestion
origins:en: en:France, fr:Dijon
```

File: taxonomies/food/categories.txt [hunk 19]
```diff
@@ -15795,7 +15778,7 @@ protected_name_type:en: pdo
 fr:Fleurie
 protected_name_file_number:en: PDO-FR-A0930
 protected_name_type:en: pdo
-origins:en: en:France, fr:Beaujolais
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:15781]:
```suggestion
origins:en: en:France, fr:Beaujolais
```

File: taxonomies/food/categories.txt [hunk 20]
```diff
@@ -16532,7 +16515,7 @@ origins:en: en:France
 fr:Juliénas
 protected_name_file_number:en: PDO-FR-A1025
 protected_name_type:en: pdo
-origins:en: en:France, fr:Beaujolais
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:16518]:
```suggestion
origins:en: en:France, fr:Beaujolais
```

File: taxonomies/food/categories.txt [hunk 21]
```diff
@@ -18286,7 +18266,7 @@ wikidata:en:Q42862923
 
 <en:Wines from France
 fr:Morgon
-origins:en: en:France, fr:Beaujolais
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:18269]:
```suggestion
origins:en: en:France, fr:Beaujolais
```

File: taxonomies/food/categories.txt [hunk 22]
```diff
@@ -18297,7 +18277,7 @@ origins:en: en:France
 
 <en:Wines from France
 fr:Moulin-à-Vent
-origins:en: en:France, fr:Beaujolais
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:18280]:
```suggestion
origins:en: en:France, fr:Beaujolais
```

File: taxonomies/food/categories.txt [hunk 23]
```diff
@@ -19726,7 +19705,7 @@ wikidata:en:Q42862538
 
 <en:Wines from France
 fr:Régnié
-origins:en: en:France, fr:Beaujolais
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:19708]:
```suggestion
origins:en: en:France, fr:Beaujolais
```

File: taxonomies/food/categories.txt [hunk 24]
```diff
@@ -19795,7 +19774,7 @@ origins:en: en:France
 <en:Wines from France
 fr:Rosé des Riceys
 wikidata:en:Q2167983
-origins:en: en:France, fr:Champagne
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:19777]:
```suggestion
origins:en: en:France, fr:Champagne
```

File: taxonomies/food/categories.txt [hunk 25]
```diff
@@ -20175,7 +20154,7 @@ protected_name_type:en: pgi
 
 <en:Wines from France
 fr:Saint-Amour
-origins:en: en:France, fr:Beaujolais
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:20157]:
```suggestion
origins:en: en:France, fr:Beaujolais
```

File: taxonomies/food/categories.txt [hunk 26]
```diff
@@ -39000,7 +38962,7 @@ description:en:Kūčiukai are small round crunchy biscuits made from wheat flour
 en:Cornish Clotted Cream
 protected_name_file_number:en: PDO-GB-0014
 protected_name_type:en: pdo
-origins:en: en:United Kingdom, en:Cornwall
+origins:en: en:United Kingdom
```
Comment (teolemon) [inline taxonomies/food/categories.txt:38965]:
```suggestion
origins:en: en:United Kingdom, en:Cornwall
```

File: taxonomies/food/categories.txt [hunk 27]
```diff
@@ -44584,7 +44530,7 @@ agribalyse_food_code:en:12119
 ciqual_food_code:en:12119
 ciqual_food_name:en:Ossau-Iraty cheese, from ewe's milk
 ciqual_food_name:fr:Ossau-Iraty
-origins:en: en:France, fr:Pays-Basque
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:44533]:
```suggestion
origins:en: en:France, fr:Pays-Basque
```

File: taxonomies/food/categories.txt [hunk 28]
```diff
@@ -45976,7 +45918,7 @@ en:Wensleydale cheese
 en:Yorkshire Wensleydale
 protected_name_file_number:en: PGI-GB-0652
 protected_name_type:en: pgi
-origins:en: en:United Kingdom, en:England, en:Yorkshire
+origins:en: en:United Kingdom
```
Comment (teolemon) [inline taxonomies/food/categories.txt:45921]:
```suggestion
origins:en: en:United Kingdom, en:Yorkshire
```

File: taxonomies/food/categories.txt [hunk 29]
```diff
@@ -54478,14 +54413,14 @@ nl:Acaciahoningen uit de Alsace
 fr:Miel de Cévennes
 protected_name_file_number:en: PGI-FR-1148
 protected_name_type:en: pgi
-origins:en: en:France, fr:Cévennes
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:54416]:
```suggestion
origins:en: en:France, fr:Cévennes
```

File: taxonomies/food/categories.txt [hunk 30]
```diff
@@ -54565,23 +54500,23 @@ fr:Miels de sapin des Vosges
...
 nl:Honingen van de Gatinais
-origins:en: en:France, fr:Gatinais
+origins:en: en:France
 
 <en:Honeys from France
 en:Honeys from Jura
 fr:Miels du Jura
 nl:Honingen uit de Jura
-# Disabled for safety: origins:en: en:France, fr:Jura
+# Disabled for safety: origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:54519]:
```suggestion
# Disabled for safety: origins:en: en:France, fr:Jura
```

File: taxonomies/food/categories.txt [hunk 31]
```diff
@@ -54565,23 +54500,23 @@ fr:Miels de sapin des Vosges
...
 fr:Miels des Pays de Loire
 nl:Honingen uit de Loirestreek
-origins:en: en:France, fr:Pays de Loire
+origins:en: en:France
 
 <en:Honeys from France
 fr:Miels du Gâtinais, Miel du Gätinais
 nl:Honingen van de Gatinais
-origins:en: en:France, fr:Gatinais
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:54513]:
```suggestion
origins:en: en:France, fr:Gatinais
```

File: taxonomies/food/categories.txt [hunk 32]
```diff
@@ -54565,23 +54500,23 @@ fr:Miels de sapin des Vosges
...
 protected_name_file_number:en: PDO-FR-0204
 protected_name_type:en: pdo
-origins:en: en:France, fr:Vosges
+origins:en: en:France
 
 <en:Honeys from France
 fr:Miels des Pays de Loire
 nl:Honingen uit de Loirestreek
-origins:en: en:France, fr:Pays de Loire
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:54508]:
```suggestion
origins:en: en:France, fr:Pays de Loire
```

File: taxonomies/food/categories.txt [hunk 33]
```diff
@@ -54565,23 +54500,23 @@ fr:Miels de sapin des Vosges
 nl:Sparrenhoningen uit de Vogezen
 protected_name_file_number:en: PDO-FR-0204
 protected_name_type:en: pdo
-origins:en: en:France, fr:Vosges
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:54503]:
```suggestion
origins:en: en:France, fr:Vosges
```

File: taxonomies/food/categories.txt [hunk 34]
```diff
@@ -54540,7 +54475,7 @@ origins:en: en:France, en:Corsica
 en:Honey of Biscay
 fr:Miels de Gascogne
 nl:Honingen uit de Gascogne
-# Disabled for safety: origins:en: en:France, fr:Gascogne
+# Disabled for safety: origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:54478]:
```suggestion
# Disabled for safety: origins:en: en:France, fr:Gascogne
```

File: taxonomies/food/categories.txt [hunk 35]
```diff
@@ -54478,14 +54413,14 @@ nl:Acaciahoningen uit de Alsace
...
-origins:en: en:France, fr:Cévennes
+origins:en: en:France
 
 <en:Acacia honeys
 <fr:Miels du Jura
 en:Acacia honeys from Jura
 fr:Miels d’acacia du Jura
 nl:Acaciahoningen uit de Jura
-# Disabled for safety: origins:en: en:France, fr:Jura
+# Disabled for safety: origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:54423]:
```suggestion
# Disabled for safety: origins:en: en:France, fr:Jura
```

File: taxonomies/food/categories.txt [hunk 36]
```diff
@@ -56625,15 +56559,15 @@ fr:Huiles d'olive de Nice
 nl:Olijfoliën uit Nice, Olijfolies uit Nice
 protected_name_file_number:en: PDO-FR-0322
 protected_name_type:en: pdo
-origins:en: en:France, fr:Nice
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:56562]:
```suggestion
origins:en: en:France, fr:Nice
```

File: taxonomies/food/categories.txt [hunk 37]
```diff
@@ -56625,15 +56559,15 @@ fr:Huiles d'olive de Nice
...
 
 <en:Olive oils from France
 en:Olive oil from Nîmes
 fr:Huile d'olive de Nîmes
 nl:Olijfoliën uit Nîmes, Olijfolies uit Nîmes
 protected_name_file_number:en: PDO-FR-0441
 protected_name_type:en: pdo
-origins:en: en:France, fr:Nîmes
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:56570]:
```suggestion
origins:en: en:France, fr:Nîmes
```

File: taxonomies/food/categories.txt [hunk 38]
```diff
@@ -66467,7 +66397,7 @@ fr:Pruneaux d'Agen
 hr:Agen suhe šljive
 protected_name_file_number:en: PGI-FR-0149
 protected_name_type:en: pgi
-origins:en: en:France, fr:Agen
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:66400]:
```suggestion
origins:en: en:France, fr:Agen
```

File: taxonomies/food/categories.txt [hunk 39]
```diff
@@ -105478,7 +105397,7 @@ tr:Siyah zeytin
 fr:Olives noires de Nyons
 protected_name_file_number:en: PDO-FR-0374
 protected_name_type:en: pdo
-origins:en: en:France, fr:Nyons
+origins:en: en:France
```
Comment (teolemon) [inline taxonomies/food/categories.txt:105400]:
```suggestion
origins:en: en:France, fr:Nyons
```

File: taxonomies/labels.txt (hunk 2 above)
Comment (teolemon) [inline taxonomies/labels.txt:3793]:
```suggestion
origins:en: en:france, fr:Hauts-de-France
```

**Review by alexgarel — APPROVED** [review]

**Pull request merged**